bin/sl-deploy.js

```diff
@@ -12,11 +13,13 @@ var fs = require('fs');
 var maybeTunnel = require('strong-tunnel');
 var path = require('path');
 
-function printHelp($0, prn) {
-  var USAGE = fs.readFileSync(require.resolve('./sl-deploy.txt'), 'utf-8')
-      .replace(/%MAIN%/g, $0)
-      .trim();
+if (path.basename(process.cwd()) === 'test')
+  GLB.setRootDir(path.dirname(process.cwd()));
 
+function printHelp($0, prn) {
+  var USAGE = (GLB.t('msgAll1') + '\n' + GLB.t('msgAll2'))
```

Comment:
due to the max number of string supported by Globalization Pipeline on Bluemix service.